lib/active_model_serializers/test/serializer.rb

```diff
@@ -0,0 +1,122 @@
+module ActiveModelSerializers
+  module Test
+    module Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+      end
+
+      # Asserts that the request was rendered with the appropriate serializers.
+      #
+      #  # assert that the "PostSerializer" serializer was rendered
+      #  assert_serializer "PostSerializer"
+      #
+      #  # return a custom error message
+      #  assert_serializer "PostSerializer", "PostSerializer not rendered"
+      #
+      #  # assert that the instance of PostSerializer was rendered
+      #  assert_serializer PostSerializer
+      #
+      #  # assert that the "PostSerializer" serializer was rendered
+      #  assert_serializer :post_serializer
+      #
+      #  # assert that the rendered serializer starts with "Post"
+      #  assert_serializer %r{\APost.+\Z}
+      #
+      #  # assert that no serializer was rendered
+      #  assert_serializer nil
+      #
+      def assert_serializer(expectation, message = nil)
+        @assert_serializer.expectation = expectation
+        @assert_serializer.message = message
+        @assert_serializer.response = response
+        assert(@assert_serializer.matches?, @assert_serializer.message)
```

Comment:
So, people using RSpec would need to define `assert(expected, actual)` and any library not using `ActionController::TestCase` would also need to include `ActiveModelSerializers::Test::Serializer`

something like:

```ruby
module AMSAssertSerializer
  extend ActiveSupport::Concern
  included do
    include ActiveModelSerializers::Test::Serializer
  end
  def assert(expected, actual)
    expect(actual).to eq(expected)
  end
end
RSpec.configuration.include AMSAssertSerializer, type: :controller
```